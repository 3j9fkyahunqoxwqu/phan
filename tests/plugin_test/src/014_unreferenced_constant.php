<?php

class A {
    const myconst = 'x';
    const unreferencedconst = 'y';
    /** @suppress PhanUnreferencedConstant testing suppression*/
    const unreferencedconst2 = 'z';
}
class B extends A {
}

$x = A::myconst;
$b = new B();
