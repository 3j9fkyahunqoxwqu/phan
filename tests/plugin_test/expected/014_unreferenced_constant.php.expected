src/014_unreferenced_constant.php:5 PhanUnreferencedConstant Possibly zero references to constant \A::unreferencedconst
